src/components/v5/common/ActionSidebar/partials/forms/ManageTokensForm/hooks.ts

```diff
@@ -52,11 +59,18 @@ export const useManageTokens = (
     defaultValues: useMemo(
       () => ({
         createdIn: Id.RootDomain.toString(),
-        selectedTokenAddresses: colonyTokens.map(({ token }) => ({
-          token: token.tokenAddress,
-        })),
+        selectedTokenAddresses: colonyTokens
+          .map(({ token }) => ({
+            token: token.tokenAddress,
+          }))
+          .sort((a) =>
+            a.token === blockchainNativeTokenAddress ||
+            a.token === colonyNativeTokenAddress
```

Comment:
Please use a proper variable name, one that's more readable, other than just `a`

Reply:
This is a sort function. What other naming do you propose? Something like `tokenA` would be ok?